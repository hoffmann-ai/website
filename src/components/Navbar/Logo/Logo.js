// @flow strict
import React from 'react';
import { withPrefix, Link } from 'gatsby';
import styles from './Logo.module.scss';

type Props = {
  img: string,
};

const Logo = ({ img }: Props) => (
  <div className={styles['logo']}>
    <Link to="/">
      <img
        src={withPrefix(img)}
        className={styles['logo__photo']}
        width="70"
        height="70"
        alt="logo"
      />
    </Link>
  </div>
);

export default Logo;
