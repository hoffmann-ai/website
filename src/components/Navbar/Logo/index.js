// @flow strict
export { default } from './Logo';
