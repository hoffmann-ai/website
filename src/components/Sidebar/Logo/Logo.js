// @flow strict
import React from 'react';
import { withPrefix, Link } from 'gatsby';
import styles from './Logo.module.scss';

type Props = {
  img: string,
};

const Logo = ({ img }: Props) => (
  <div className={styles['logo']}>
    <Link to="/">
      <img
        src={withPrefix(img)}
        className={styles['logo__photo']}
        width="75"
        height="75"
        alt="logo"
      />
    </Link>
  </div>
);

export default Logo;
