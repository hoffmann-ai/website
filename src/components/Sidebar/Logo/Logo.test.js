// @flow strict
import React from 'react';
import renderer from 'react-test-renderer';
import Logo from './Logo';

describe('Logo', () => {
  const props = {
    img: '/logo.png',
    isIndex: false
  };

  it('renders correctly', () => {
    const tree = renderer.create(<Logo {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
