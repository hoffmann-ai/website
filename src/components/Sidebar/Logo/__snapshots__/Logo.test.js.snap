// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<div
  className="logo"
>
  <a
    href="/"
  >
    <img
      alt="logo"
      className="logo__photo"
      height="75"
      src="/logo.png"
      width="75"
    />
  </a>
</div>
`;
